fastlane/Fastfile

```diff
@@ -265,6 +269,17 @@ platform :android do
                 end
 
             end
+
+            validateReleaseNotes(releaseNotes: releaseNotes)
```

Comment:
NIT: we are setting `500` to be used here. The thing is it is possible that the release notes is 500 chars but it will fail in `release_notes_playstore` since we are adding more strings to the release notes there.

I think we can keep this check since it allows us to fail early but i think the more important check is in  `release_notes_playstore`